Evaluation/apply_training.py

```diff
@@ -0,0 +1,107 @@
+import os
+import sys
+import json
+import git
+from tqdm import tqdm
+
+import uproot
+import numpy as np
+import pandas as pd
+from tensorflow.keras.models import load_model
+
+import mlflow
+import hydra
+from hydra.utils import to_absolute_path
+from omegaconf import DictConfig, OmegaConf
+
+sys.path.insert(0, "../Training/python")
+from common import setup_gpu
+
+@hydra.main(config_path='.', config_name='apply_training')
+def main(cfg: DictConfig) -> None:
+    # set up paths & gpu
+    mlflow.set_tracking_uri(f"file://{to_absolute_path(cfg.path_to_mlflow)}")
+    path_to_artifacts = to_absolute_path(f'{cfg.path_to_mlflow}/{cfg.experiment_id}/{cfg.run_id}/artifacts/')
+    setup_gpu(cfg.gpu_cfg)
```

Comment:
What parameters should be set to run inference on CPU-only?

Reply:
yes, now it can be done by setting `gpu_cfg: null` in the yaml config file